### PR TITLE
feature(Vector): Normalize vector to 0-1

### PR DESCRIPTION
**Pull request by szemyd**
Closes #

### CODE REVIEW

**Comment by szemyd** [comment]
Can be achieved by [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html)
